[IMP] Employees: Update equipment (#13061)
odoo/documentation: content/applications/hr/employees/equipment.rst

Click the :icon:`fa-cubes` :guilabel:`Equipment Count` smart button, and all equipment currently
assigned to the employee appears in individual Kanban cards.

> Each Kanban card displays the name and model of the equipment on the first line, then the serial
> number, and last, the employee's name. Any current maintenance requests appear at the bottom of the

Reviewer: ```suggestion
Each Kanban card displays the equipment's name and model on the first line, followed by the serial number and, lastly, the employee’s name. Any current maintenance requests appear at the bottom of the
```

> number, and lastly, the employee's name. Any current maintenance requests appear at the bottom of the

Reviewer 2: let's add a note that the serial number is optional (:

employee <employees/equipment>`.

.. note::

>    It does not matter what employee is selected, and if they have any equipment assigned to them or
>    not. This is only used to get to the :guilabel:`Equipment` list.

Reviewer: ```suggestion
   It does not matter what employee is selected, or whether they have any equipment assigned to them. This step is only used to get to the :guilabel:`Equipment` list.
```

In the Kanban view of the employee's equipment, clear the default :guilabel:`Assigned Employee`
filter in the search bar. This presents *all* equipment in the database, including those assigned to
individual employees and whole departments.

Click into the search bar, and select :guilabel:`Employee` in the :icon:`oi-group` :guilabel:`Group
By` column. The equipment is now organized in a Kanban view, by employee.

In the Kanban view displaying all employee equipment records, equipment can be reassigned by

> clicking and dragging an equipment card to the desired employee, to change ownership of the
> equipment.

Reviewer: ```suggestion
clicking and dragging an equipment card to the desired employee. This changes the ownership of the
equipment.
```
